ocs/defaults.py

```diff
@@ -32,6 +32,7 @@
 CEPH_IMAGE = "ceph/ceph:v14.2.0-20190410"
 ROOK_IMAGE = "rook/ceph:master"
 DEPLOYMENT_PLATFORM = 'AWS'
+TEST_NAMESPACE = 'ocs-qe'
```

Comment:
I would say it's ok to keep it here. We don't have a guideline about namespace usage in test code (eg. I would like to see namespace fixture with default function scope), but it's clear that tests should create k8s objects in custom test namespace.

Reply:
@mbukatov that makes sense. If multiple tests are utilizing this and potentially overriding the value which is propagated to other tests then I can see it being here. I just want to avoid having simple variables get put here when they really belong inside the test script themselves.